docs/research/nextjs-runtime-rendering.md

```
@@ -0,0 +1,862 @@
+### What it is called now
+
+In `next@16.3.1` there is no separate PPR flag. `experimental.ppr` and the
+`experimental_ppr` route segment export were **removed** in 16.0, and Partial
+Prerendering is now a consequence of enabling Cache Components:
+
+> Next.js 16 removes the experimental Partial Prerendering (PPR) flag and configuration
+> options, including the route level segment `experimental_ppr`. Starting with Next.js 16,
+> you can opt into PPR using the `cacheComponents` configuration.
+>
+> — [Version 16 upgrade guide, "Partial Prerendering (PPR)"](https://nextjs.org/docs/app/guides/upgrading/version-16#partial-prerendering-ppr)
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)